pom.xml

```diff
@@ -163,7 +163,7 @@
     <dependency>
       <groupId>org.apache.hive</groupId>
       <artifactId>hive-jdbc</artifactId>
-      <version>2.0.0</version>
+      <version>3.1.1</version>
```

Comment:
I wouldn't upgrade to this verison of Hive as it's potentially backwards incompatible with previous versions of Hive and we're not using it in production anywhere.